packages/medusa/src/loaders/helpers/routing/__fixtures__/server/index.ts

```diff
@@ -3,19 +3,24 @@ import {
   ModulesDefinition,
   registerMedusaModule,
 } from "@medusajs/modules-sdk"
-import { ContainerRegistrationKeys } from "@medusajs/utils"
-import { asValue, createContainer } from "awilix"
+import {
+  ContainerRegistrationKeys,
+  createMedusaContainer,
+} from "@medusajs/utils"
+import { asValue } from "awilix"
 import express from "express"
 import jwt from "jsonwebtoken"
 import { MockManager } from "medusa-test-utils"
 import querystring from "querystring"
 import supertest from "supertest"
 import apiLoader from "../../../../api"
+import { getResolvedPlugins } from "../../../../helpers/resolve-plugins"
 import featureFlagLoader, { featureFlagRouter } from "../../../../feature-flags"
 import passportLoader from "../../../../passport"
 
 import RoutesLoader from "../.."
 import { config } from "../mocks"
+import { MedusaContainer } from "medusa-core-utils"
```

Comment:
todo: I suggest to use the one from the types package, this one should be removed

Reply:
👍